internal/cli/atlas_clusters_list.go

```diff
@@ -27,22 +27,25 @@ type atlasClustersListOpts struct {
 	*globalOpts
 	pageNum      int
 	itemsPerPage int
-	store        store.ClusterLister
+	storeAT      store.ClusterLister
```

Comment:
there's no need to create a new store, just create a new interface that defines the two methods you need